grammar.js

```diff
@@ -616,16 +614,12 @@ module.exports = grammar({
                 seq(OPR().not, after_not),
                 seq(
                     OPR().minus,
-                    alias(
-                        seq(
-                            // ensure the expression immediately follows the
-                            // opening paren
-                            token.immediate(BRACK().open_paren),
-                            $.pipeline,
-                            BRACK().close_paren
-                        ),
-                        $.expr_parenthesized
-                    )
+                    seq(
+                        // ensure the expression immediately follows the
+                        // opening paren
+                        token.immediate(BRACK().open_paren),
+                        inline_pipeline($, BRACK().close_paren),
```

Comment:
not quite related to your pr, but i just found a bug

```nushell
(let x = 4; x) 
```

is rare but valid nu, so we should probably fix that
```suggestion
                        $._top_level_block,
                        BRACK().close_paren
```

Reply:
I run test with this change locally and encounter an error, maybe we should leave this for later?

Reply:
yeah, i agree. this wasn't part of your pr